spec/features/admin/customers_spec.rb

```diff
@@ -148,6 +136,24 @@
             expect(page).to have_content "$0.00"
           end
         end
+
+        # this adds the additional payments to an order, and checks for customer balance
+        it "updates customer balance with additional negative payments, on an order" do
+          payment2
+
+          # verifies the payment2 was correctly added to the order
```

Comment:
This feels a bit redundant to me given the line `expect... to have_content` below. In a dev's mind, `expected` always reads as a verification. It may even be more explicit if we replace `$-25.00` with `payment2.amount`? 

In specs, there is always a very thin boundary between hardcoding or referencing a var. There's no golden rule IMO.

Reply:
Ok, switching to referencing the value, instead of hardcoding. Also, removing the unnecessary comment.